src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private int index = new Random().nextInt(Colors.values().length);
```

Comment:
here the variable 'index' is initialized only once and accordingly the method always returns the same value. Let's make only the Random constant, write the name in capital letters and the logic in the middle of the method